universal_data_permissions_scanner/cli.py

```diff
@@ -59,14 +59,35 @@ def main(ctx: click.Context, debug: bool, out: str, out_format: str):
 @main.command()
 @click.pass_context
 @click.option('--username', '-u', required=True, type=str, help="Username")
-@click.option('--password', '-p', required=True, type=str, help="Password")
+@click.option('--password', '-p', required=False, type=str, help="Password")
+@click.option('--rsa-key', '-r', required=False, type=click.File('r'), help="Path to RSA private key")
+@click.option('--rsa-pass', required=False, type=str, help="RSA key password")
 @click.option('--account', '-a', required=True, type=str, help="Account")
 @click.option('--host', '-t', required=False, type=str, help="Hostname")
 @click.option('--warehouse', '-w', required=False, type=str, help="Warehouse")
-def snowflake(ctx: click.Context, username: str, password: str, account: str, host: str, warehouse: str):
+def snowflake(
+    ctx: click.Context,
+    username: str,
+    password: str,
```

Comment:
I think it also change the password to `Optional[str]` no?

Reply:
yes, changing